test/copy_template_test.go

```diff
@@ -127,6 +127,7 @@ var _ = Describe("Copy template task", func() {
 					SourceTemplateName: testtemplate.RhelTemplateName,
 					TargetTemplateName: NewTemplateName,
 					Template:           testtemplate.NewRhelDesktopTinyTemplate().Build(),
+					SetOwnerReference:  "true",
```

Comment:
Can you add a test for when it is disabled?

Reply:
added